packages/ra-core/src/dataProvider/index.ts

```diff
@@ -45,7 +45,5 @@ export {
     useCreate,
     useDelete,
     useDeleteMany,
-    useRefreshWhenVisible,
     withDataProvider,
-    useIsAutomaticRefreshEnabled,
```

Comment:
Note for self: uin the EditController, pass an option to the useGetOne call to disable auto refresh on the record data, as it creates unexpected effects on forms.